src/test/scala/com/atomist/rug/test/gherkin/GherkinRunnerTest.scala

```diff
@@ -7,21 +7,26 @@ import com.atomist.project.archive.RugArchiveReader
 import com.atomist.rug.TestUtils
 import com.atomist.rug.runtime.js.JavaScriptContext
 import com.atomist.rug.ts.TypeScriptBuilder
-import com.atomist.source.{ArtifactSource, ArtifactSourceUtils, SimpleFileBasedArtifactSource, StringFileArtifact}
+import com.atomist.source.{ArtifactSource, SimpleFileBasedArtifactSource, StringFileArtifact}
+import gherkin.ast.ScenarioDefinition
 import jdk.nashorn.api.scripting.{NashornScriptEngine, NashornScriptEngineFactory}
 import org.scalatest.{FlatSpec, Matchers}
-import org.springframework.expression.spel.ast.AstUtils
 
 class GherkinRunnerTest extends FlatSpec with Matchers {
 
   import GherkinReaderTest._
 
   "Gherkin runner" should "fail without JS" in {
+    val el = new TestExecutionListener
     val as = SimpleFileBasedArtifactSource(TwoScenarioFeatureFile)
-    val grt = new GherkinRunner(new JavaScriptContext(as))
+    val grt = new GherkinRunner(new JavaScriptContext(as), Option.empty, Seq(el))
```

Comment:
I think we are using None, Option[type] and Some(thing)....as a stylistic thing.